src/DSharp.Compiler/ScriptModel/Expressions/ExpressionType.cs

```diff
@@ -32,5 +32,6 @@ internal enum ExpressionType
         InlineScript = 22,
         NewDelegate = 23,
         Object = 24,
+        ObjectInitilizer = 25
```

Comment:
Collection initializer and Object initializer are different things

Reply:
also, typo

Reply:
I'll differentiate when I implement collection initializers, however the parser doesn't handle it very well :(